rundeckapp/grails-app/domain/rundeck/ScheduledExecution.groovy

```diff
@@ -328,8 +328,12 @@ class ScheduledExecution extends ExecutionContext {
             se.uuid = data.uuid
         }
         se.timeout = data.timeout?data.timeout.toString():null
-        se.retry = data.retry?data.retry.toString():null
-        se.retryDelay = data.retryDelay?data.retryDelay.toString():null
+        if(data.retry instanceof Map){
+            se.retry = data.retry.retry?data.retry.retry.toString():null
```

Comment:
just a style comment: but you can do this: 

~~~{.groovy}
se.retry = data.retry.retry?.toString() ?: null
~~~